src/components/custom-bottom-nav-bar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Container, Nav, Navbar} from 'react-bootstrap';
+import logo192 from '../assets/images/logo192.png';
+
+class CustomBottomNavbar extends React.Component {
+
+    render() {
+        return (
+            <>
+                <Navbar bg="light" variant="light" sticky={"bottom"}>
+                    <Container>
+                        <Navbar.Brand href="/">
+                            <img
```

Comment:
Looks not so aligned yet. Let's have another PR to fix it unless it's easy to fix right now.

Reply:
I want to make a custom navbar without using bootstrap for the bottom but didn't have the time to do it yet.

Reply:
OK. Let's just leave it.